src/test/java/org/dita/dost/IntegrationTest.java

```diff
@@ -346,6 +346,16 @@ public void testconref_topiconly() throws Throwable {
                 .test();
     }
 
+    @Test
+    public void testconref_to_specialization() throws Throwable {
+        builder().name("conref_to_specialization")
+                .transtype(XHTML)
```

Comment:
We could save a bit of time on the integration test process if we only run `PREPROCESS` (since this is testing something that's only part of preprocessing) -- or possibly just add this conref test into an existing integration test for conref. I'm not sure if any of the existing tests are a good fit though.

Reply:
I just used an existing conref test as a model. They seemed to be very narrow in their focus.

Limiting it to preprocess makes sense though.